### PR TITLE
chore(yarn): upgrade yarn to 1.22.17

### PR DESCRIPTION
**Pull request by jeffchew**
### Related Ticket(s)

No related issue

### Description

We are seeing connectivity issues with `yarn install` on our CI, this is an attempt to see if this might help at all.

### Changelog

**Changed**

- Upgrade yarn to `1.22.17`

### CODE REVIEW

File: .yarnrc [hunk 1]
```diff
@@ -2,8 +2,8 @@
 # yarn lockfile v1
 
 
-version v1.19.0
-lastUpdateCheck 1597082923381
+version "1.22.10"
```
Comment (emyarod) [inline .yarnrc:5]: is this supposed to match up with the release version in `yarn-path`?

**Review by emyarod — COMMENTED** [review]
just 1 question about the version

File: .yarnrc (hunk 1 above)
Comment (jeffchew) [reply .yarnrc:5]: hmm yeah, this was auto generated and I set the policy to use 1.22.17. Not sure why it says 1.22.10...

Comment (emyarod) [reply .yarnrc:5]: gotcha, I guess it should be fine since it was autogenerated and still the same minor version

**Review by emyarod — APPROVED** [review]
looks good to me if the version mismatch is just from yarn's autogenerating!

**Pull request merged**